pymc/logprob/scan.py

```diff
@@ -395,19 +395,14 @@ def find_measurable_scans(fgraph, node):
         # Get any `Subtensor` outputs that have been applied to outputs of this
         # `Scan` (and get the corresponding indices of the outputs from this
         # `Scan`)
-        output_clients: list[tuple[Variable, int]] = sum(
-            [
-                [
-                    # This is expected to work for `Subtensor` `Op`s,
-                    # because they only ever have one output
-                    (cl.default_output(), i)
-                    for cl, _ in fgraph.get_clients(out)
-                    if isinstance(cl.op, Subtensor)
-                ]
-                for i, out in enumerate(node.outputs)
-            ],
-            [],
-        )
+        output_clients: list[tuple[Variable, int]] = [
```

Comment:
@ricardoV94 I changed the other two, but would you like me to change this one too? It's a bit different

Reply:
This one seems fine. The sum was never needed if I'm decoding it correctly?